LASzip/src/laswriteitemraw.hpp

```diff
@@ -245,11 +246,11 @@ class LASwriteItemRaw_POINT14_LE : public LASwriteItemRaw
       ((LAStempWritePoint14*)buffer)->number_of_returns = ((LAStempWritePoint10*)item)->number_of_returns;
       ((LAStempWritePoint14*)buffer)->scan_angle = I16_QUANTIZE(((LAStempWritePoint10*)item)->scan_angle_rank/0.006f);
     }
-
-    *((F64*)&buffer[22]) = ((LAStempWritePoint10*)item)->gps_time;
```

Comment:
Why "avoid unaligned write of F64 to buffer" ... ? You mean "force unaligned write of F64 to buffer" ... ?!?

Reply:
Some older ARM-devices did not support unaligned memory access like that. I'm unsure if any correct platform suffer from that. Also it is a performance issue (memcpy does not solve that). The warning from the compiles is there for those two reasons.

Reply:
Dereferencing misaligned pointers is undefined behaviour in C++.  (Although it does work on Intel CPUs)
See: [clang UndefinedBehaviorSanitizer](https://clang.llvm.org/docs/UndefinedBehaviorSanitizer.html)